envoy/rds/config_traits.h

```diff
@@ -54,7 +55,9 @@ class ConfigTraits {
    * to the corresponding route configuration class.
    * @throw EnvoyException if the new config can't be applied of.
    */
-  virtual ConfigConstSharedPtr createConfig(const Protobuf::Message& rc) const PURE;
+  virtual ConfigConstSharedPtr createConfig(const Protobuf::Message& rc,
+                                            Server::Configuration::ServerFactoryContext& context,
+                                            bool validate_clusters_default) const PURE;
```

Comment:
Please document the new params in the Doxygen-style comments above.

Reply:
done. thanks.